### DISCUSSION TITLE
Schema changes in patch versions?

### DISCUSSION THREAD

**Discussion by MaximilianFranz** [Dlubal-Software/RFEM_Python_Client#405] [Q&A]
Hey all, 

we've noticed that in the results returned from `ResultTables.LinesSupportForces` the key of `node_no` was changed to `node_no` from `node_number` in a recent **patch** update of the RFEM 6.07 Webervice server.

This broke our scripts and took quite some time to figure out.

Where can we read up on such changes? And shouldn't breaking API changes be limited to major and minor versions?

Thanks already, 
Max

**Comment by jlubojacka** [comment]
Hello, 

we also found out some time ago (June 2024 - not sure about patch program version) that the key of comment column in `ResultTables.MembersLocalDeformations` was changed from `section` to `specification`. We had a hard time figuring out the cause of problems. After that, we disabled automatic updates to make sure the code corresponds with result tables, but that's not an ideal solution.

I agree that these breaking changes of RFEM program should be listed somewhere. 
Jana

**Comment by MaximilianFranz** [comment]
@jarabroz - can you bring this into the team as a recommendation? :)
